test/cpp/demos/main.cpp

```diff
@@ -0,0 +1,27 @@
+#include <iostream>
+#include <string>
+#include <torch/csrc/jit/mobile/import.h>
+#include <ATen/core/ivalue.h>
+#include <torch/csrc/autograd/generated/variable_factories.h>
+
+int main(int argc, char* argv[]) {
+    std::cout << "lite_interpreter_demo is starting..." << std::endl;
+
+    if (argc == 2) {
+        std::string file_path = argv[1];
+        std::vector<c10::IValue> inputs;
+
+        std::cout << "Loading model from: " << file_path << std::endl;
+        torch::jit::mobile::Module bc = torch::jit::_load_for_mobile(file_path);
+
+        inputs.push_back(torch::ones({1, 1, 4, 4}));
```

Comment:
It's a hard-coded input, which means it may not work for generic mobile models. I lean to make this a test instead of a cli executable.

Reply:
That's a good point. Let me see how to add a unit test for it.

Reply:
A unit test is added.